cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -431,6 +440,14 @@ public UUID userTaskId() {
       return _queryParams;
     }
 
+    private EndPoint endPoint() {
```

Comment:
Would we make the getter public? -- i.e. even though we are not using it from outside the class, we might potentially want to do so later.